src/components/Table/Table.tsx

```diff
@@ -20,12 +21,15 @@ export const TableAction = (props: {
       size="small"
       variant={props.variant}
       sx={{ p: "9px 12px" }}
-      css={{ whiteSpace: "nowrap" }}
+      css={{
+        whiteSpace: "nowrap",
+      }}
       onClick={(e) => {
         e.stopPropagation()
         e.preventDefault()
         props.onClick?.()
       }}
+      data-intersect={props["data-intersect"]}
```

Comment:
```suggestion
      {...props}
```
spread the buttons props instead